lib/logmsg/logmsg-serialize.c

```diff
@@ -37,7 +37,7 @@
 #define OLD_LMM_REF_MATCH 0x0001
 
 static void
-_setup_ts_processed(LogStamp *timestamps, const LogStamp *processed)
+_setup_ts_processed(UnixTime *timestamps, const UnixTime *processed)
```

Comment:
We think we should use the original name (LogStamp) instead, and should not do this renaming.

unixtime is a widely used term that includes only seconds. It sounds strange to include timezone info and microseconds. However, these concepts fit well with the LogStamp concept.